learned.lic

```diff
@@ -27,7 +27,7 @@ class LearnedData
     args = parse_args(arg_definitions)
 
     survival_skills = ['Evasion', 'Athletics', 'Perception', 'Stealth', 'Locksmithing', 'Thievery', 'First Aid', 'Outdoorsmanship', 'Skinning']
-    lore_skills = ['Alchemy', 'Appraisal', 'Enchanting', 'Forging', 'Mechanical Lore', 'Performance', 'Scholarship', 'Tactics']
+    lore_skills = ['Alchemy', 'Appraisal', 'Enchanting', 'Forging', 'Mechanical Lore', 'Performance', 'Scholarship', 'Tactics', 'Outfitting']
```

Comment:
Should Engineering also be added?

Reply:
Yes. I should have caught. Doh!. I can add. One moment.